nextcloud/calendar
# Use linkify directive from @nextcloud/vue

Replace local linkify directive with the one from @nextcloud/vue.

## How to test?

1. Create a new event with a description containing a link. 
2. Share the calendar via a public link and open it in an incognito tab. 
3. The link in the description should be clickable.
